homeassistant/components/sensor/deconz.py

```diff
@@ -61,7 +62,8 @@ def async_update_callback(self, reason):
         """
         if reason['state'] or \
            'reachable' in reason['attr'] or \
-           'battery' in reason['attr']:
+           'battery' in reason['attr'] or \
+           'daylight' in reason['attr']:
```

Comment:
This shouldn't be necessary since daylight is part of state, the or part is only for parameters in config

Reply:
Gotcha. Will remove.